themes/beaver/assets/css/foundations/css-variables.css

```
@@ -1,8 +1,15 @@
 /* CSS Variables Foundation - WP1.1 */
 /* Extracted from 330 occurrences across 44 CSS files */
 /* 2025-10-12: CSS Migration Project 2509 */
+/* 2026-04-14: Updated to Obsidian Ruby design system (Stitch) */
+
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@400;500;600;700;800&family=Inter:wght@300;400;500;600;700&display=swap');
```

Stylelint (17.6.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Manrope:wght@400;500;600;700;800&family=Inter:wght@300;400;500;600;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Manrope:wght@400;500;600;700;800&family=Inter:wght@300;400;500;600;700&display=swap'" (import-notation)

(import-notation)